feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/ui/QuoteCard.kt

```diff
@@ -0,0 +1,210 @@
+package com.hedvig.android.feature.changeaddress.ui
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.expandVertically
+import androidx.compose.animation.shrinkVertically
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.hedvig.android.core.designsystem.newtheme.SquircleShape
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.android.core.ui.R
+import com.hedvig.android.core.ui.UiMoney
+import com.hedvig.android.feature.changeaddress.data.Address
+import com.hedvig.android.feature.changeaddress.data.AddressId
+import com.hedvig.android.feature.changeaddress.data.MoveIntentId
+import com.hedvig.android.feature.changeaddress.data.MoveQuote
+import kotlinx.datetime.LocalDate
+import octopus.type.CurrencyCode
+
+@Composable
+fun QuoteCard(
+  movingDate: String?,
+  quote: MoveQuote,
+  onExpandClicked: () -> Unit,
+  isExpanded: Boolean,
+) {
+  Column(
+    modifier = Modifier
+      .padding(horizontal = 16.dp)
+      .clip(SquircleShape)
+      .clickable { onExpandClicked() }
+      .background(shape = SquircleShape, color = Color(0xFFF0F0F0))
+      .padding(16.dp),
+  ) {
```

Comment:
Since this is a card, it *may* be a better fit to have the top level composable be a HedvigCard, with the right clickable lambda and so on, which will also follow our theme well enough to not have to do any custom colors, and then have the column inside of it, without having to clip it or anything like that